patches/chrome-BUILD.gn.patch

```diff
@@ -65,20 +76,41 @@ index 960921acd7ebb0ba99092745e8d30bca76821f2a..534fe8a987d939b2dc24c8cf20d4e950
  
    group("chrome") {
      deps = [
-@@ -593,6 +610,12 @@ if (is_win) {
-       "--scm=1",
-       "--bundle_id=$chrome_mac_bundle_id",
+@@ -595,10 +616,32 @@ if (is_win) {
      ]
+   }
+ 
++  if (brave_chromium_build) {
++    brave_tweak_info_plist("brave_app_plist") {
++      _chrome_app_plist_output = get_target_outputs(":chrome_app_plist")
```

Comment:
I tried to move this blocks in the `//brave/BUILD.gn`
But, I got this complain.
```
ERROR at //brave/BUILD.gn:361:34: Target not found in this context.
      _chrome_app_plist_output = get_target_outputs(":chrome_app_plist")
                                 ^-------------------------------------
//brave:chrome_app_plist
was not found. get_target_outputs() can only be used for targets
previously defined in the current file.
```

Reply:
you won't be able to use `get_target_outputs` anymore so you can just reference the actual output

Reply:
`get_target_outputs` only works within the build file

Reply:
If we can add "brave_app_plist" target under brave/, it would be the best.
However, it makes we use internal implementation detail and it breaks the rule of chrome's templates. 
IMO, adding "brave_app_plist" target in chrome/BUILD.gn is not bad for rebase because it is inserted to a separate block.

Reply:
so, I guess what I was thinking is that there isn't a lot of benefit to using the `chrome` and `chrome_app` targets. They are small and simple and replacing them with targets in Brave will probably makes things easier moving forward. wdyt?

Reply:
Only `chrome_app` and `chrome_app_plist` are moved to //brave because there are many targets depends on `//chrome`.